tko_hr_pt/hr_payroll_pt.py

```diff
@@ -0,0 +1,950 @@
+
+    }
+
+class hr_employee(osv.osv):
+    '''
+    Employee
+    '''
+    
+    _history_vals_log = (
+                         {'job_id':'Cargo'}, 
+                         {'children':'Numero de filhos'},
+                         {'marital':'Estado Civil'},
+                         {'handicap':'Incapacitado'},
+                         {'military':'Militar'},
+                         {'manager':'Gerente'},
+                         {'ownership':'Titularidade IRS'},
+                         {'medic_exam':'Exame Medico'}
+                         )
+    
+    _ownership_transl = (
+                         {'unique':'Casado, unico titular'},
+                         {'double':'Casado, dois titulares'},
+                         {'none':'Solteiro'},
+                         )
+    
+    _marital_transl = (
+                       {'single' : 'Solteiro'}, 
+                       {'married' : 'Casado'}, 
+                       {'widower' : 'Viuvo'}, 
+                       {'divorced' : 'Divorciado'}
+                       )
+    _ownership = {'unique':'Casado, unico titular',
+                  'double':'Casado, dois titulares',
+                  'none':'Solteiro'}
+                         
+    
+    _marital = {'single' : 'Solteiro', 
+                'married' : 'Casado', 
+                'widower' : 'Viuvo', 
+                'divorced' : 'Divorciado'}
+    
+    _inherit = 'hr.employee'
+    _name = 'hr.employee'
+    _description = 'Employee'
+
+    _columns = {
+        'handicap': fields.boolean('Handicap', help="If is considered handicap by law"),
+        'handicap_degree': fields.selection([('under_60','Under 60%'),('equal_60', 'Equal or over 60% and under 80%'),('equal_80','Equal or over 80%')], 'Handicap Degree'),
+        'ownership': fields.selection([('unique','Unique'),('double','Double'),('none', 'Não definido')],'Ownership',select=True, required=True),
+        'military': fields.boolean('Military', help="If is military"),
+        'ssnid': fields.char('SS Identification', size=64, help="Social Security Identification of employee"),
+        'history_lines': fields.one2many('hr.employee.history', 'employee_id', 'History Lines' , readonly=True),
+        'income_type': fields.selection([
+                                         ('A - Rendimentos do trabalho dependente','A - Rendimentos do trabalho dependente'),
+                                         ('B - Rendimentos empresariais e profissionais', 'B - Rendimentos empresariais e profissionais'),
+                                         ('E - Rendimentos de capitais', 'E - Rendimentos de capitais'),
+                                         ('F - Rendimentos prediais', 'F - Rendimentos prediais'),
+                                         ('G - Incrementos patrimoniais', 'G - Incrementos patrimoniais'),
+                                         ('H - Pensões', 'H - Pensões'),
```

Comment:
It would be wiser for the selection  identifiers to be just the code ("A", "B", ...) ...